pkg/registry/registry.go

```diff
@@ -262,6 +262,7 @@ func (r PlainRegistry) listArchsWithAuth(ctx context.Context, client http.Client
 				log.DefaultLogger.WithContext(ctx).Printf("failed to get pointed manifest for arch %s of %s/%s:%v. Skipping\n", manifest.Platform.Architecture, registry, image, err)
 				continue
 			}
+			req.Header.Set("Accept", "application/vnd.oci.image.manifest.v1+json")
```

Comment:
There are 2 `Content-Type` possible value sin this block. Shouldn't we accept both?

Reply:
Thanks for spotting.
I just added the docker distribution one